frontend/src/components/module-viewer/content/ContentBlock.tsx

```diff
@@ -1,38 +1,30 @@
-import { Box, Divider, Flex, useDisclosure, VStack } from "@chakra-ui/react";
+import {
+  Box,
+  Center,
+  Divider,
+  Flex,
+  useDisclosure,
+  VStack,
+} from "@chakra-ui/react";
 import React, { useContext, useState } from "react";
 import { Draggable } from "react-beautiful-dnd";
 
+import EditorContext from "../../../contexts/ModuleEditorContext";
+import RenderComponents from "./helpers/ContentBlockTable";
 import { ContentBlockState } from "../../../types/ContentBlockTypes";
-import { TextBlock, ImageBlock, VideoBlock } from "../../common/content";
+
+import ColumnBlock from "./blocks/column/ColumnBlock";
 import DeleteModal from "../../common/DeleteModal";
 import EditContentOptionsMenu from "../EditContentOptionsMenu";
-
 import { ReactComponent as DragHandleIconSvg } from "../../../assets/DragHandle.svg";
-import { EditTextModal, EditImageModal, EditVideoModal } from "./modals";
-import createContentBlockRenderers, {
-  EmptyConfigEntry as Empty,
-} from "../../common/content/ContentBlockRenderer";
+import { EmptyConfigModal } from "./helpers/ContentBlockTableUtils";
 
-import EditorContext from "../../../contexts/ModuleEditorContext";
-
-const [CONTENT_BLOCKS, EDIT_MODALS] = createContentBlockRenderers({
-  column: Empty,
-  heading: Empty,
-  text: {
-    renderBlock: TextBlock,
-    renderEditModal: EditTextModal,
-  },
-  link: Empty,
-  button: Empty,
-  image: {
-    renderBlock: ImageBlock,
-    renderEditModal: EditImageModal,
+// Create content block table, but add column since it defaults to Empty
+const [CONTENT_BLOCKS, EDIT_MODALS] = RenderComponents({
+  column: {
+    renderBlock: ColumnBlock,
+    renderEditModal: EmptyConfigModal,
   },
```

Comment:
Is this empty just because the button isn't displayed? Or is it displayed?

Reply:
It's empty because there is no edit modal for the column component. The edit modals are dealt with by the `ColumnContent` component internally.